src/test/groovy/com/jakewharton/sdkmanager/internal/PackageResolverTest.groovy

```diff
@@ -229,6 +265,24 @@ class PackageResolverTest {
   }
 
   @FixtureName("outdated-android-m2repository")
+  @Test public void outdatedIndirectSupportLibraryRepositoryIsDownloaded() {
+    project.apply plugin: 'com.android.application'
+
+    def localMaven = new File(fixture.getRoot(), "localMaven")
+    assertThat(localMaven).exists()
+    project.repositories.maven {
+      url = localMaven
```

Comment:
To avoid making internet connection at the tests. I have created localMaven repo with nested library.
Bad thing I had to copy-paste it through "up-to-date", "missing", "outdated" fixtures.